### DISCUSSION TITLE
iOS Home Bar

### DISCUSSION THREAD

**Discussion by PatriRoman** [Telefonica/mistica-design#1942] [Proposals]
### Goal

Necesitamos tener la Home Bar de iOS como elemento único separado de la Tab Bar.

### Context

Para poder diseñar las páginas internas de las aplicaciones en donde no existe la Tab Bar pero si la Home Bar necesitamos que añadáis este elemento por separado, ya que ahora mismo no existe y va siempre ligado a la Tab Bar.

### Impact

Añadir un nuevo elemento al sistema de diseño de Home Bar para iOS

### Discussion Questions

La prioridad no es alta porque de momento podemos trampear el diseño como hicimos hasta ahora :)

**Comment by yceballost** [comment]
Buenas Patri!

Vamos a incluir esto que comentas en las librerías y aprovecharemos para actualizar alguna que otra cosa de nativo

Hicimos issue #1944

**Reply by PatriRoman** [reply]
Genial gracias!
